domaindriven-core/src/DomainDriven/Persistance/Postgres/Internal.hs

```diff
@@ -122,12 +126,19 @@ createRetireFunction conn =
           \language plpgsql;"
 
 simplePool' :: MonadUnliftIO m => PG.ConnectInfo -> m (Pool Connection)
-simplePool' connInfo =
-    createPool (liftIO $ PG.connect connInfo) (liftIO . PG.close) 1 5 5
+simplePool' connInfo = simplePool (PG.connect connInfo)
 
 simplePool :: MonadUnliftIO m => IO Connection -> m (Pool Connection)
-simplePool getConn =
-    createPool (liftIO getConn) (liftIO . PG.close) 1 5 5
+simplePool getConn = do
+    --  Using stripesAndResources because the default is crazy:
+    --  "Set num resources to cores and crash if there are fewer stripes"
```

Comment:
Could you add a link to https://github.com/scrive/pool/pull/16 to add some context?

Reply:
Ah and the comment is backwards. It's 'set striped to num cores and crash if there are fewer resources than stripes'.